tools/wptrunner/docs/conf.py

```diff
@@ -12,8 +12,8 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
-import os
+#import sys
+#import os
```

Comment:
I kept these around because the whole file is just the Sphinx generated config with defaults commented out (such as https://github.com/gsnedders/web-platform-tests/blob/f14a312ef5c4a6650c4899c194c8f43d252b295c/tools/wptrunner/docs/conf.py#L21) where `sys`/`os` used. I think either we should comment them out or drop way more of the file?